go/private/go_toolchain.bzl

```diff
@@ -93,4 +100,26 @@ go_toolchain_flags = rule(
     attrs = {
         "compile_flags": attr.string_list(mandatory=True),
     },
-)
\ No newline at end of file
+)
+
+def _external_linker_impl(ctx):
+  cpp = ctx.fragments.cpp
+  features = ctx.features
+  options = cpp.compiler_options(features)
```

Comment:
Nit: I think this would be more readable as one big statement using + instead of several statements with +=.

Reply:
Done